firmware/src/networking/packets.rs

```diff
@@ -0,0 +1,19 @@
+use crate::utils::Reliable;
+use firmware_protocol::PacketType;
+
+/// Packets is an accessor to internal logic <-> network messaging system
+pub struct Packets {
+	/// The latest `Message` that should be sent
+	pub serverbound: Reliable<PacketType>,
```

Comment:
Can be done. I  generally prefer to use `client/serverbound` because they are absolute terms and not relative. Often there is shared networking code between clients and servers so in a commons crate "inbound" and "outbound" do not make sense. Here it is fine however

Reply:
your point is fair. We can keep it if you want.

Another problem with inbound/outbound is that its unclear if its from the perspective of the async task, or the perspective of the firmware as a whole. So I think I like your way better